docs/planning/backend/app/db/__init__.py

```
@@ -0,0 +1,3 @@
+from app.db.database import engine, SessionLocal, init_db, get_db, get_db_context
+
+__all__ = ["engine", "SessionLocal", "init_db", "get_db", "get_db_context"]
```

Ruff (0.14.10)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)